packages/protocol/contracts/governance/Validators.sol

```diff
@@ -345,30 +341,27 @@ contract Validators is
    * @dev Fails if the account does not have sufficient weight.
    */
   function registerValidator(
-    string calldata name,
     bytes calldata publicKeysData
   )
     external
     nonReentrant
     returns (bool)
   {
     require(
-      bytes(name).length > 0 &&
       // secp256k1 public key + BLS public key + BLS proof of possession
       publicKeysData.length == (64 + 48 + 96)
     );
     // Use the proof of possession bytes
     require(checkProofOfPossession(publicKeysData.slice(64, 48 + 96)));
 
-    address account = getLockedGold().getAccountFromActiveValidator(msg.sender);
+    address account = getAccounts().getAccountFromActiveValidationSigner(msg.sender);
```

Comment:
Should we require that the account has registered a DataEncryptionKey?

Reply:
We will no longer use data encryption keys for attestation purposes, so I'm not sure we need to require them?

Reply:
Oh, definitely no need then